layouts/partials/example-go-grpc.html

```diff
@@ -10,50 +10,45 @@
 {{ $majorVersion := int (index $version 0) }}
 {{ $minorVersion := int (index $version 1) }}
 {{ $patchVersion := int (index $version 2) }}
-{{ $versionGe105 := or (eq $currentBranch "master") (gt $majorVersion 1) (and (eq $majorVersion 1) (gt $minorVersion 0)) (and (eq $majorVersion 1) (eq $minorVersion 0) (ge $patchVersion 5)) }}
+{{ $versionGe110 := or (eq $currentBranch "master") (gt $majorVersion 1) (and (eq $majorVersion 1) (gt $minorVersion 1)) (and (eq $majorVersion 1) (ge $minorVersion 1) (ge $patchVersion 0)) }}
```

Comment:
Does this mean that all pre-1.0.5 documentation pages will get broken with this change?
I'm not saying it's a bad thing, but should be mentioned at least in the commit message